pkg/sdk/event_tables_def.go

```diff
@@ -0,0 +1,148 @@
+package sdk
+
+import g "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/poc/generator"
+
```

Comment:
See 
https://docs.snowflake.com/en/sql-reference/sql/create-event-table#usage-notes 
and
https://docs.snowflake.com/en/developer-guide/logging-tracing/event-table-operations#operations-supported-on-an-event-table
There are more operations than listed here (like drop or truncate)

Reply:
true but you can use the normal tables SDK for that. DROP TABLE is not part of the event tables SDK